bazel/api-gen/rendering/entities.ts

```diff
@@ -162,3 +163,26 @@ export interface ParameterEntry {
   isOptional: boolean;
   isRestParam: boolean;
 }
+
+export interface FunctionWithOverloads {
+  name: string;
+  signatures: FunctionEntry[];
+  implementation: FunctionEntry | null;
+}
+
+export interface InitializerApiFunctionEntry extends DocEntry {
+  callFunction: FunctionWithOverloads;
+  subFunctions: FunctionWithOverloads[];
+
+  __adevMetadata__?: {
```

Comment:
can this be `__docsMetadata__` so that its not tied directly to adev?

Reply:
good idea. I'm good with this